index.ts

```diff
@@ -163,14 +163,48 @@ collect.set('isNotifications', [
 
 export const isOrganizationProfile = (): boolean => exists('meta[name="hovercard-subject-tag"][content^="organization"]');
 
+export const isOrganizationRepoList = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/repositories$/.test(getCleanPathname(url));
+collect.set('isOrganizationRepoList', [
+	'https://github.com/orgs/github/repositories',
+	'https://github.com/orgs/github/repositories?page=2',
+]);
+
 export const isOrganizationRepo = (): boolean => Boolean(document.querySelector<HTMLElement>('[data-owner-scoped-search-url]')?.dataset['ownerScopedSearchUrl']!.startsWith('/org'));
 
-export const isOrganizationDiscussion = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/teams\/[^/]+($|\/discussions)/.test(getCleanPathname(url));
-collect.set('isOrganizationDiscussion', [
+export const isOrganizationPackagesList = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/packages$/.test(getCleanPathname(url));
+collect.set('isOrganizationPackagesList', [
+	'https://github.com/orgs/github/packages',
+]);
+
+export const isOrganizationPeopleList = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/people$/.test(getCleanPathname(url));
```

Comment:
```suggestion
export const isOrganizationMembers = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/people$/.test(getCleanPathname(url));
```

There's no "single member" page so it doesn't need "list"